### ISSUE TITLE
SelfDuration off in ubuntu/unix?

### ISSUE THREAD

**Issue by fflaten** [nohwnd/Profiler#18]
I'm seeing some weird results in a devcontainer based on ubuntu 20.04. SelfDuration is reporting 1min in a 1 seconds run. 
Only me or unix-compatibility issue? Tried with both PS 7.0.6 and 7.1.3 in same image. Works fine on Windows using PS 7.1.3.

```powershell
PS /workspaces/Profiler> $trace = Trace-Script { Start-Sleep -Seconds 1 }

Running in PowerShell 7.0.6.
Tracing...
Run  finished after 00:00:01.0016429
Tracing done. Got 5 trace events.
Processing 5 trace events. 
Loading sources for 1 files.
Figuring out flow.
Sorting trace events to file lines.
Figuring out durations per line.
Counting averages and percentages.
Getting Top50 that take most percent of the run.
Getting Top50 with the longest average Duration.
Getting Top50 with the longest Duration.
Getting Top50 with the longest average SelfDuration.
Getting Top50 with the longest SelfDuration.
Getting Top50 with the most hits.
Progress: A: 00:01:40.0548300 (0 ms) -> A: 00:01:40.0540700 (-1 ms) -> A: 00:01:40.0669400 (13 ms)
Done. Try $trace.Top50 | Format-Table to get the report. There are also Top50Duration, Top50Average, Top50SelfDuration, Top50SelfAverage, Top50HitCount, All, Events and Files.

PS /workspaces/Profiler> [Profiler.Tracer]::Hits | ft SelfDuration, Text -AutoSize

SelfDuration     Text
------------     ----
00:00:00.0264400 & $ScriptBlock
00:00:00.0007600 {
00:01:40.0373800 Start-Sleep -Seconds 1
00:00:00.0066600 }
00:00:00.0194100 Set-PSDebug -Trace 0
```

**Comment by nohwnd** [comment]
Did not try it on anything other than 64 bit windows powershell 5 and 7. We will see when tests are written. And CI setup.

**Comment by nohwnd** [comment]
Yup I can see it on CI build now. https://github.com/nohwnd/Profiler/pull/21

**Comment by nohwnd** [comment]
$t = { [System.Diagnostics.Stopwatch]::GetTimestamp() }; $t1= &$t;Start-Sleep -milli 10; $t2=&$t;[timespan]::fromticks($t2-$t1).TotalMilliseconds
1033.65 <- it is 100 times higher

**Comment by nohwnd** [comment]
Fixed in #22
